resource-managers/kubernetes/core/pom.xml

```diff
@@ -108,6 +108,14 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlet</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-http</artifactId>
+    </dependency>
```

Comment:
I'm not fully following here -- why does this change to the logging watcher require adding explicit dependencies on jetty ?  Or are these separate changes?